setup.py

```diff
@@ -11,5 +11,5 @@
       packages=['open_catalyst_dataset'],
       scripts=[],
       include_package_data=True,
-      install_requires=['pymatgen','ase'],
+      install_requires=['pymatgen','ase', 'catkit (version 0.5.4)'],
```

Comment:
Should this be `catktit (version 0.5.4)` or something like `catkit=0.5.4`?

Edit:  Looks like newer commits address this.